### ISSUE TITLE
windows compile failed for 0.10.0

### ISSUE THREAD

**Issue by Xudong-Huang** [rust-bitcoin/rust-secp256k1#45]
**invalid numeric argument '/Wno-unused-function'**

```sh
Compiling secp256k1 v0.10.0
error: failed to run custom build command for `secp256k1 v0.10.0`
process didn't exit successfully: `D:\project\rust\rust-trustnote\target\debug\build\secp256k1-e50cad406cbac261\build-script-build` (exit code: 101)
--- stdout
TARGET = Some("x86_64-pc-windows-msvc")
OPT_LEVEL = Some("0")
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
TARGET = Some("x86_64-pc-windows-msvc")
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CC_x86_64-pc-windows-msvc = None
CC_x86_64_pc_windows_msvc = None
HOST_CC = None
CC = None
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CFLAGS_x86_64-pc-windows-msvc = None
CFLAGS_x86_64_pc_windows_msvc = None
HOST_CFLAGS = None
CFLAGS = None
DEBUG = Some("true")
running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.13.26128\\bin\\HostX64\\x64\\cl.exe" "/nologo" "/MD" "/Z7" "/I" "depend/secp256k1/" "/I" "depend/secp256k1/include" "/I" "depend/secp256k1/src" "-g" "-Wno-unused-function" "/DSECP256K1_BUILD=1" "/DUSE_NUM_NONE=1" "/DUSE_FIELD_INV_BUILTIN=1" "/DUSE_SCALAR_INV_BUILTIN=1" "/DUSE_FIELD_10X26=1" "/DUSE_SCALAR_8X32=1" "/DUSE_ENDOMORPHISM=1" "/DENABLE_MODULE_ECDH=1" "/DENABLE_MODULE_RECOVERY=1" "/W4" "/FoD:\\project\\rust\\rust-trustnote\\target\\debug\\build\\secp256k1-7d587a2033c27679\\out\\depend/secp256k1/contrib\\lax_der_parsing.o" "/c" "depend/secp256k1/contrib/lax_der_parsing.c"
cargo:warning=cl : Command line error D8021 : invalid numeric argument '/Wno-unused-function'
exit code: 2

--- stderr
thread 'main' panicked at '

Internal error occurred: Command "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.13.26128\\bin\\HostX64\\x64\\cl.exe" "/nologo" "/MD" "/Z7" "/I" "depend/secp256k1/" "/I" "depend/secp256k1/include" "/I" "depend/secp256k1/src" "-g" "-Wno-unused-function" "/DSECP256K1_BUILD=1" "/DUSE_NUM_NONE=1" "/DUSE_FIELD_INV_BUILTIN=1" "/DUSE_SCALAR_INV_BUILTIN=1" "/DUSE_FIELD_10X26=1" "/DUSE_SCALAR_8X32=1" "/DUSE_ENDOMORPHISM=1" "/DENABLE_MODULE_ECDH=1" "/DENABLE_MODULE_RECOVERY=1" "/W4" "/FoD:\\project\\rust\\rust-trustnote\\target\\debug\\build\\secp256k1-7d587a2033c27679\\out\\depend/secp256k1/contrib\\lax_der_parsing.o" "/c" "depend/secp256k1/contrib/lax_der_parsing.c" with args "cl.exe" did not execute successfully (status code exit code: 2).

', C:\Users\DELL\.cargo\registry\src\github.com-1ecc6299db9ec823\gcc-0.3.54\src\lib.rs:1670:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by apoelstra** [comment]
Is this still happening?

It may be a bug in `cc` upstream. If it's still happening we can simply remove the `-Wno-unused-function` flag on Windows though.

**Comment by apoelstra** [comment]
Can you try compiling https://github.com/rust-bitcoin/rust-secp256k1/pull/61 ?

**Comment by alekseysidorov** [comment]
And some other build log from appveyor ^_^

https://ci.appveyor.com/project/alekseysidorov/btc-transaction-utils/build/1.0.47#L150

**Comment by apoelstra** [comment]
Should be fixed by #61
